README.md

```diff
@@ -1,70 +1,92 @@
-# Getting Started with Create React App
+![swrpgc banner](https://user-images.githubusercontent.com/17935770/131261369-477cff64-39e5-462a-81bf-3f55944e512e.png)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+This application acts as a companion for the D&D game: **Star Wars: Edge of the Empire**. The companion app allows you to keep track of and store your in-game characters. You may also connect and communicate with your friends and party up so that everyone can witness the dice rolls that will unfold the story in their legendary campaign.
 
-In the project directory, you can run:
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+![feaures](https://user-images.githubusercontent.com/17935770/131262008-e0fb73fa-3967-41bb-852d-6fd1f71b45a7.png)
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+ - User can register an account
+ - User may create a new character that contains their information, characteristics, features
+ - All of your characters can be seen in one place
+ - Dice rolls can be be configured with actual dice animation
+ - User and friends can see a live dice log and the dice rolls
```

Comment:
Should we separate these last ones out as stretch goals for now, since they aren't part of the MVP?

Reply:
You right! This mine as well be updated as we go anyways!